pkg/common/common.go

```diff
@@ -173,7 +179,17 @@ func GetOpenAPITypeFormat(typeName string) (string, string) {
 	if !ok {
 		return "", ""
 	}
-	return mapped[0], mapped[1]
+	return mapped.name, mapped.format
+}
+
+// Returns the zero-value for the given type along with true if the type
+// could be found.
+func GetOpenAPIZeroValue(typeName string) (interface{}, bool) {
```

Comment:
s/Get//

Reply:
I don't know, the function above is called `GetOpenAPITypeFormat`, I just followed the same pattern.

Reply:
My standard comment when I see GetFoo in golang code: the idiomatic function name for those mappings without state is `Foo`.

Reply:
Yes, the question is: do we want to be correct or consistent? :-)